.travis.yml

```diff
@@ -7,6 +7,7 @@ python:
 env:
   global:
     - DISPLAY=:99.0
+    - secure: "bYe6WOTAnlS8Ru4ODWSSOnHffxcN23NkKZh4M0eO510HvZGCMB4zZn8afiVKGXd1YqsoRfMXTBZJ0yBcFEvWnyH7S4kd+7d1PpNS4kgLVKtLCW5d7Wc5GA6uh1jWLS+zKFBNN5sZ8OVc7rCsLCBRDEoI94wBKYwDX2Kk1WKylz8="
```

Comment:
Just confirming: as secure variables aren't available to forks, this isn't going to create failures on Travis based tests on our olympia forks?

Reply:
absolutely not, in theory that script will only be run if run via a cron - see https://github.com/mozilla/addons-server/pull/6243/files#diff-354f30a63fb0907d4ad57269548329e3R77